packages/components/src/components/bal-popover/bal-popover.sass

```diff
@@ -7,6 +7,16 @@
   +modifier(active)
     .bal-popover__content
       display: block
+  +modifier(hint)
+    .bal-popover__content
+      +mobile
+        min-height: 100vh
+        .bal-popover__content__inner
+          min-height: 100vh
+      +tablet
+        max-width: 320px
+      +desktop
+        max-width: 464px
```

Comment:
why exactly 464px, should we use some relative sizing unit here instead of px maybe?